Source/WebKit/UIProcess/mac/WKTextAnimationManager.mm

```diff
@@ -92,7 +92,14 @@ - (void)addTextAnimationForAnimationID:(NSUUID *)uuid withData:(const WebCore::T
 
     switch (data.style) {
     case WebCore::TextAnimationType::Initial:
-        effect = adoptNS([PAL::alloc_WTSweepTextEffectInstance() initWithChunk:chunk.get() effectView:_effectView.get()]);
+        if (!_webView->isWritingToolsTextReplacementsFinished()) {
```

Comment:
Let's try to move this check to the web process, and avoid attempting to re-add the animation entirely.